common.xml: Add commands, capability and flag for Moving Target Indicators (MTI)

nexton-winjeel opened the pull request:
This PR adds two commands, a capability flag and a status flag for Moving Target Indicators (MTI). 

There a number of camera systems that can provide Moving Target Indicators (MTI) on the video stream, where objects that are moving in the scene are highlighted. Some examples:
* https://www.youtube.com/watch?v=IrDRrjEZJM0
* https://vimeo.com/948485078

Currently marked as work-in-progress (WIP).

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -2255,6 +2255,18 @@
         <description>Stops ongoing tracking.</description>
         <param index="1" label="Target Camera ID" minValue="0" maxValue="255" increment="1">Target camera ID. 7 to 255: MAVLink camera component id. 1 to 6 for cameras attached to the autopilot, which don't have a distinct component id. 0: all cameras. This is used to target specific autopilot-connected cameras. It is also used to target specific cameras when the MAV_CMD is used in a mission.</pa ...
       </entry>
+      <entry value="2020" name="MAV_CMD_CAMERA_START_MTI" hasLocation="false" isDestination="false">
```
These would go into development.xml.

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -2255,6 +2255,18 @@
+      <entry value="2020" name="MAV_CMD_CAMERA_START_MTI" hasLocation="false" isDestination="false">
+        <wip/>
+        <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>Enable Moving Target Indicators (MTI).</description>
```
Probably 
```suggestion
        <description>Enable Moving Target Indicators (MTI) on streamed video.</description>
```

hamishwillee reviewed (commented):
@rmackay9 @julianoes @auturgy Any comment?

This looks good to me other than it is in common rather than development.xml.

nexton-winjeel commented:
Thanks @hamishwillee. I've updated as follows:
* Added the missing `bitmask="true"` to `CAMERA_TRACKING_STATUS_FLAGS`;
* Moved the commands into `development.xml`; and
* Updated the descriptions of the commands per your comments.

hamishwillee reviewed (commented):
Thanks! @auturgy @rmackay9 @julianoes - I'll add to next week's call, but would appreciate comment about whether you have objections or suggestions.

hamishwillee reviewed (approved):
@nexton-winjeel We discussed in the MAV call - makes sense/no objections. 

Just a few comments:
1. For future reference, this "kind of thing" looks like it might have some overlap with the RAS-A definitions. So if you do end up going down those paths, we might want to copy from https://github.com/Dronecode/air-iop-definitions/blob/master/message_definitions/v1.0/ras_a.xml
2. This is WIP. As soon as you have a working implementation, please create a PR to move definitions into common and remove WIP. Attach evidence this is working for you. We just want verification that there is an implementation and that this has been tested properly. 

Merging.
